domains/classroom.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jOaawd",
+        "email": "jorti28@apps.pusd.org"
+    },
+    "record": {
+        "CNAME": "gv-pfibmgoyouk6yz.dv.googlehosted.com"
```

Comment:
I cannot reach your site
"This site can’t be reached" `DNS_PROBE_FINISHED_NXDOMAIN`